### DISCUSSION TITLE
[STUDIO][1.2.0] Touches raccourcies ?

### DISCUSSION THREAD

**Discussion by Raphael-zoreille** [witsa/synapps#664] [Ideas]
Bonjour, 

La mise en place de touches raccourcies est-elle prévue? 

- Zoom / dézoom
- Retour à 100% ou valeur prédéfinie
- Retour page affichée largueur fenêtre de travail....
- etc...
Merci

**Comment by Raphwit** [comment]
Bonjour, effectivement cela peut être des fonctionnalités intéressantes pour de futures versions, nous prenons note, merci ! 😊
